renku/core/models/calamus.py

```diff
@@ -82,3 +118,86 @@ def _deserialize(self, value, attr, data, **kwargs):
 
 
 fields.Uri = Uri
+
+
+class Nested(fields.Nested):
+    """Nested field that passes along client and commit info."""
+
+    def __init__(self, *args, propagate_client=True, **kwargs):
+        """Init method."""
+        super().__init__(*args, **kwargs)
+
+        self.propagate_client = propagate_client
+
+    @property
+    def schema(self):
+        """The nested Schema object.
```

Comment:
When you generate docs, we could perserve links for code browsing by:
```suggestion
        """The nested ``marshmallow.Schema`` object.
```